.gitmodules

```diff
@@ -2,3 +2,7 @@
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
 	branch = v1.3.0
+[submodule "tests/integration/ef_tests"]
+	path = tests/integration/ef_tests/testdata
+	url = https://github.com/ethereum/tests
+  shallow = true
```

Comment:
shallow preserves less git history, meaning it is faster to clone and fetch. mirrored from our good friends at go-eth:

https://github.com/ethereum/go-ethereum/blob/master/.gitmodules#L4